tests/cloudsdk_tests/test_cloud.py

```diff
@@ -0,0 +1,95 @@
+"""
+About: It contains some Functional Unit Tests for CloudSDK and to run and test them on per unit level
+"""
+
+import pytest
+import sys
+
+if 'cloudsdk_tests' not in sys.path:
+    sys.path.append(f'../../libs/cloudsdk')
+from cloudsdk import CloudSDK
+from configuration_data import TEST_CASES
+
+
+@pytest.mark.run(order=1)
+@pytest.mark.bridge
+@pytest.mark.nat
+@pytest.mark.vlan
+@pytest.mark.cloud_connect
+class TestCloudSDK(object):
+
+    @pytest.mark.sdk_version_check
+    def test_cloud_sdk_version(self, instantiate_cloudsdk, instantiate_testrail, instantiate_project):
+        print("1")
+        cloudsdk_cluster_info = {}  # Needed in Test Result
+        try:
+            response = instantiate_cloudsdk.portal_ping()
+
+            cloudsdk_cluster_info['date'] = response._commit_date
+            cloudsdk_cluster_info['commitId'] = response._commit_id
+            cloudsdk_cluster_info['projectVersion'] = response._project_version
+            instantiate_testrail.update_testrail(case_id=TEST_CASES["cloud_ver"], run_id=instantiate_project,
+                                                 status_id=1, msg='Read CloudSDK version from API successfully')
+            PASS = True
+        except:
+            cloudsdk_cluster_info = {'date': "unknown", 'commitId': "unknown", 'projectVersion': "unknown"}
+            instantiate_testrail.update_testrail(case_id=TEST_CASES["cloud_ver"], run_id=instantiate_project,
+                                                 status_id=0, msg='Could not read CloudSDK version from API')
+            PASS = False
+
+        assert PASS, cloudsdk_cluster_info
+
+
+@pytest.mark.run(order=2)
+@pytest.mark.bridge
+@pytest.mark.nat
+@pytest.mark.vlan
+@pytest.mark.firmware
+class TestFirmware(object):
+
+    @pytest.mark.firmware_create
+    def test_firmware_create(self, upload_firmware, instantiate_testrail, instantiate_project):
+        print("2")
```

Comment:
Clean up all spurious print messages.

Reply:
cleaned up the print() statements that are suspicious